### DISCUSSION TITLE
Unable to use it no code generated

### DISCUSSION THREAD

**Discussion by PGTBoos** [community/community#19497] [Copilot Conversations]
So I installed github copilot, due to poor documentation, I missed that I had to past a device key (and put it first in notepad before use), because a second time launch VS studio or you install it again it wont generate a key anymore and so now I'm without key without trial period.
I was really curious about this, to bad.

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
